src/ucp/api/ucp.h

```diff
@@ -3135,6 +3108,169 @@ ucs_status_ptr_t ucp_worker_flush_nb(ucp_worker_h worker, unsigned flags,
                                      ucp_send_callback_t cb);
 
 
+/**
+ * @brief UCP active message rendezvous parameters field mask.
+ *
+ * The enumeration allows specifying which fields in @ref ucp_params_t are
+ * present. It is used to enable backward compatibility support.
+ */
+enum ucp_am_params_field {
+  /* Flafs field */
+  UCP_AM_FIELD_FLAGS      = UCS_BIT(0),
+  /* Max size of the iovec returned by the initial AM */
+  UCP_AM_FIELD_IOVEC_SIZE = UCS_BIT(1)
+};
+
+/**
+ * @brief Tuning parameters for the UCP active message.
+ *
+ * The structure defines the parameters that are used for the
+ * UCP active message function
+ */
+typedef struct ucp_am_params {
+  uint64_t field_mask ;
```

Comment:
plz remove extra whitespace before `;` here and below

Reply:
Done